modules/bookDesplay.js

```diff
@@ -0,0 +1,40 @@
+// eslint-disable-next-line no-unused-vars
```

Comment:
- [ ] Kindly remove all `Eslint-disable` statements from this file. `Eslint` ensures we write clean and understandable codes and using a disable statement defeats this purpose. Kindly remove all `eslint-disable` statements in the file 👍